common/lib/src/realm_common_base.dart

```diff
@@ -16,23 +16,59 @@
 //
 ////////////////////////////////////////////////////////////////////////////////
 
+///@nodoc
+late String _noDoc;
```

Comment:
seems a debug left over?

Reply:
This is something I've added for discussion. I put this code here as workaround in order to prevent these slashes to appear in the doc API //////////. Is there some way I can do it?